_docs/concepts/handling-failures.md

```diff
@@ -0,0 +1,76 @@
+---
+category: Concepts
+title: Handling Failures
+
+parent: Traffic Management
+order: 30
+
+bodyclass: docs
+layout: docs
+type: markdown
+---
+
+Envoy sidecar/proxy provides a set of out-of-the-box _opt-in_ failure recovery features
```

Comment:
Can just say 'Envoy' rather than 'Envoy sidecar/proxy'. The deployment topology is sufficient covered in Concepts (or it should be)